forte/base_classes/active_space_solver.h

```diff
@@ -130,6 +130,13 @@ class ActiveSpaceSolver {
     /// Set if read wave function from file as initial guess
     void set_read_initial_guess(bool read_guess) { read_initial_guess_ = read_guess; }
 
+    /// @return the the label of the ActiveSpaceMethod used by this solver
+    std::string method() const;
+
+    std::shared_ptr<MOSpaceInfo> mo_space_info() const;
```

Comment:
If we don't want this memory address to change, we might want to mark this ptr const. Same below.